includes/compat/subscriptions/trait-wc-payment-gateway-wcpay-subscriptions.php

```diff
@@ -706,4 +706,34 @@ public function restore_order_pay_var() {
 			$wp->query_vars['order-pay'] = $this->order_pay_var;
 		}
 	}
+
+	/**
+	 * Add card mandate options parameters to the order payment intent if needed.
+	 * Only required for the subscription creation for card issued in India.
+	 * More details https://wp.me/pc4etw-ky
+	 *
+	 * @param WC_Order $order The order.
+	 * @return array Params to be included or empty array.
+	 */
+	public function maybe_add_mandate_to_order_payment( WC_Order $order ):  array {
+		$result = [];
+
+		if ( $this->is_subscriptions_enabled() ) {
+			$subscriptions = wcs_get_subscriptions_for_order( $order->get_id() );
+			$subscription  = reset( $subscriptions );
+			if ( $subscription ) {
+				$result['setup_future_usage']                                = 'off_session';
```

Comment:
Appreciate any input here. I think we should apply it to every order with subscriptions, and probably not here.